executor/mock/jobrunner.go

```diff
@@ -181,14 +188,40 @@ func NewJobRunner() *JobRunner {
 	}
 
 	jobRunner := &JobRunner{
-		ctx: ctx, cancel: cancel,
+		ctx:             ctx,
+		cancel:          cancel,
 		runner:          r,
 		shutdownChannel: make(chan struct{}),
 	}
 
 	return jobRunner
 }
 
+// DockerPull does a docker pull via DockerRuntime
+func DockerPull(imgName string, version string) (bool, error) {
```

Comment:
Dumb question, any reason this code isn't in standalone_test? Are there any private methods that need to be accessed?

Reply:
One nicety of putting it in standalone test, is you can pass testing.T to it, and then use assertify.

Reply:
Nope - was just trying to follow convention.  It looked like all of the tests were in `standalone_test.go`, and all of the helpers were in `jobrunner.go`.